slider/beatmap.py

```diff
@@ -146,6 +151,7 @@ def event_type(self):
     def parse(cls, start_time, event_params):
         try:
             filename, x_offset, y_offset = event_params
+            filename = cls.strip_quotes_from_filename(filename)
```

Comment:
probably not worth its own method when you can write `filename.strip(")`

Reply:
Good catch